addons/controllers/csi/vspherecsiconfig_controller.go

```diff
@@ -198,3 +204,25 @@ func (r *VSphereCSIConfigReconciler) reconcileVSphereCSIConfigNormal(ctx context
 
 	return ctrl.Result{}, nil
 }
+
+func (r *VSphereCSIConfigReconciler) getVsphereCluster(ctx context.Context,
```

Comment:
suggestion(non-blocking): Maybe create a good-first issue to make this function as an util ? I see CPI related changes in above have the same.

Reply:
Sure. Resolving since this is non-blocking.